packages/material-ui-lab/src/SelectableGroup/SelectableGroup.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as utils from './utils';
+import SelectableGroupContext from './SelectableGroupContext';
+
+function SelectableGroup(props) {
+  const [selected, setSelected] = React.useState(null);
+  const { additional, children, exclusive, value: valueProp } = props;
+
+  const createReducer = handler => {
```

Comment:
What about
```suggestion
  const createAction = handler => {
```

Reply:
It wouldn't match terminology from flux architecture. This is not creating a reducer. The returned function takes an event (the action) and the previosu state (value) and returns a new value. Maybe reverse the order of arguments from the returned function (`event, value` -> `value, event`) to make this more obvious.

Reply:
I don't mind this. Personally I don't care about the event so as the second argument I can choose not to pass it in and ignore the event returned.